src/Transposit.ts

```diff
@@ -62,6 +62,54 @@ export class Transposit {
     }
   }
 
+  private async makeCall(
```

Comment:
mmmm this is a problem. I will look into other solutions in a different diff. I just thought we'd forgotten to mark these as private.

Reply:
I mentally had it in the awkward position of "publicly accessible private API"; I.e. we have to leave it available, but don't document it, suggest people use it, or commit to not break code that uses it.

I had the idea that we could have a separate "requester" object that encapsulates the information needed to make the request, and rather than passing the Transposit object to the stash etc. we could pass just that object. It seemed like a heavyweight solution at the time, but if we really care about leaving these public, it should work.

I feel concerned, though, that the code compiles with this change.